lambdas/eventbrite/config.js

```diff
@@ -13,6 +13,7 @@ const eventsParams = page =>
     "location.longitude": -6.762739,
     "location.latitude": 54.6425126, // Cookstown
     "location.within": "60mi", // 60 mile radius (all of Northern Ireland)
+    expand: "logo,venue,organizer,format,category,subcategory,bookmark_info,refund_policy,ticket_availability",
```

Comment:
Nice! `ticket_availability` could be really useful down the line 👍

Reply:
@apoclyps Yea! On the Meetup.com side, there's `"rsvp_limit"`, `"yes_rsvp_count"` and `"waitlist_count"` - I'm hoping we can end up with some standard way of exposing when events have limited spaces (and also when they cost money)